hivemq-edge/src/frontend/src/extensions/datahub/designer/operation/OperationNode.utils.ts

```diff
@@ -94,9 +95,23 @@ export function checkValidityTransformFunction(
     ]
   }
 
-  // TODO[19497] This should not have to happen on the client side!
-  const formattedScriptName = (functionNode: Node<FunctionData>): string => {
-    return `fn:${functionNode.data.name}:latest`
+  const { transform } = operationNode.data.formData as unknown as OperationData.DataHubTransformType
+  const defaultOrder = transform.length ? transform : scriptNodes.map((e) => e.data?.id)
+
+  const allTransformScripts: DryRunResults<PolicyOperation, Script>[] = []
+  for (const scriptId of defaultOrder) {
+    const script = scriptNodes.find((e) => e.data?.id === scriptId)
```

Comment:
suggestion(non-blocking): let's use `scriptNode` instead of `e` to define what the iteratee is

```suggestion
    const script = scriptNodes.find((scriptNode) => scriptNode.data?.id === scriptId)
```